### DISCUSSION TITLE
Better Burning - Elenai interaction: putting out onFire

### DISCUSSION THREAD

**Discussion by Foreck1** [ACGaming/UniversalTweaks#354] [Tweak Suggestions]
A part of the **Better Burning** entity tweak: 
If **Elenai Dodge 2** is installed, if the player is on fire, a configurable chance (30% default) for any dodge to put out the onFire status. By default, this tweak is off.

**Comment by ACGaming** [comment]
Implemented in a431532561e4976d912241d7af164f7db1ec5df4
